presentation-opportunities.md

```diff
@@ -0,0 +1,14 @@
+# Presentation Opportunities
+The following table includes upcoming opportunities to present SPDX.
+
+If anyone see any opportunities, please add it to the table.
+
+If anyone would like to volunteer to submit a paper, please add your name.
```

Comment:
Who is this addressed to? Everyone who looks at this repo? A selected set of "approved speakers"?
I'd remove this line until we figure it out.